IdentityServer/Quickstart/Account/ExternalUserInfo.cs

```diff
@@ -0,0 +1,87 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+
+namespace IdentityServer.Quickstart.Account
+{
+    /// <summary>
+    /// The viewmodel for /connect/userinfo
+    /// </summary>
+    public class ExternalUserInfo
+    {
+        /// <summary>
+        /// Gets or sets the sub.
+        /// </summary>
+        /// <value>
+        /// The sub.
+        /// </value>
+        [JsonProperty("sub")]
+        public string sub { get; set; }
+        /// <summary>
+        /// Gets or sets the name.
+        /// </summary>
+        /// <value>
+        /// The name.
+        /// </value>
+        [JsonProperty("name")]
+        public string Name { get; set; }
+        /// <summary>
+        /// Gets or sets the name of the family.
+        /// </summary>
+        /// <value>
+        /// The name of the family.
+        /// </value>
+        [JsonProperty("family_name")]
+        public string FamilyName { get; set; }
+        /// <summary>
+        /// Gets or sets the name of the given.
+        /// </summary>
+        /// <value>
+        /// The name of the given.
```

Comment:
The name of the given :)
```suggestion
        /// Gets or sets the given name of the external user.
        /// </summary>
        /// <value>
        /// The given name of the external user.
```